src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -427,7 +434,7 @@ String tagString(final String[] tags) {
      */
     @Override
     public void count(final String aspect, final long delta, final String... tags) {
-        send(new StringBuilder(prefix).append(aspect).append(":").append(delta).append("|c").append(tagString(tags)).toString());
+        send(new StringBuilder(prefix).append(sanitiseAspect(aspect)).append(":").append(delta).append("|c").append(tagString(tags)).toString());
```

Comment:
Could we factorize this logic in a function instead of repeating it almost everywhere?